package.json

```
@@ -8,7 +8,8 @@
   },
   "main": "app.js",
   "scripts": {
-    "build": "npx @tailwindcss/cli -i ./css/styles.css -o ./dist/output.css --watch"
+    "build": "npx @tailwindcss/cli -i ./css/styles.css -o ./dist/output.css --watch",
+    "start": "node ./src/index.js"
```

Also, you could include the build script in the start script. The advantage is that anytime you run start, we want to build the styling and get the latest changes.

```suggestion
    "start": "npm run build && node ./src/index.js"
```